### PR TITLE
Recommendation for Unit test run at GH action

### PR DESCRIPTION
**Pull request by fatihayyildiz**
Github action checks workflow created
unit test command moved to checks workflow file

### CODE REVIEW

**Comment by fatihayyildiz** [comment]
@oyo I added new workflow file, runs at PR creation and checking unit tests. Could you please take a look ?

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Build Library and Portal
+        run: yarn build
```
Comment (oyo) [inline .github/workflows/checks.yml:21]: The full build takes up to 5 minutes with creating that license file and storybook which are not needed for unit tests. I suggest to only build the shared component library at this step by `yarn build:lib`

**Review by oyo — CHANGES_REQUESTED** [review]

File: .github/workflows/checks.yml (hunk 1 above)
Comment (oyo) [reply .github/workflows/checks.yml]: I committed that change and the pipeline is now down to 3 minutes which should be okay

**Review by oyo — APPROVED** [review]
Reduced run time by removing some non-essential steps. Also now only runs when PR is created or updated, not on every PR comment because that feels like too many invocations.

**Comment by fatihayyildiz** [comment]
Thanks @oyo ,I missed somehow your comments. Looks nice, I think it's ready to merge 👍

**Comment by oyo** [comment]
> Thanks @oyo ,I missed somehow your comments. Looks nice, I think it's ready to merge 👍

sure - just merge it :)

**Pull request merged**